### ISSUE TITLE
Password entry form gives silent authentication failure on long passwords

### ISSUE THREAD

**Issue by boredsquirrel** [fedora-infra/noggin#1394]
I created a 47 digit alphanumerical password and also use TOTP.

When entering the password, I get an "authentication failure". But resetting the password via email works, it is reset to that password without error.

Now switching to a more complex but only 22 digits password, login works again.

**Comment by AJCxZ0** [comment]
I have a Fedora login with OTP which is consistently working on https://accounts.fedoraproject.org but not on https://id.fedoraproject.org which is used for identification and authentication on most or all Fedora services such as [Discussion](https://discussion.fedoraproject.org/),  [Mailing Lists](https://lists.fedoraproject.org/), and [Pagure](https://pagure.io/fedora-infrastructure/issues).

For example, from https://pagure.io/fedora-infrastructure/issues I clicked \[Log in\], was sent to https://pagure.io/login/?next=https://pagure.io/fedora-infrastructure/issues were my password manager enters

- Username: `ajcxz0`
- Password: `zjdpdxgAmYgo.lPwwCrEjDCRthjUpGq9MH6pvqkx-ZsimLHiStfI89OtjiPxAu.UDJmOJtzUWbiCtmLf!zYIb.vIgtwEj9`  - Not my actual password, but equivalent.
- One-Time-Password (if you have one): `917311`

and I clicked \[Log in\]. After a moment's consideration, a red banner appeared near the top of the page containing the error, "Authentication failure".

After logging out of all Fedora sites, deleting all site data and reloading each of the pages (or visiting the service Incognito), I clicked \[Log in\], filled in credentials on [Pagure](https://pagure.io/), and arrived at https://id.fedoraproject.org/login?ipsilon\_transaction\_id=\[...\]

> 401 - Unauthorized
> Transaction expired, or cookies not available
> [Try to login again](https://id.fedoraproject.org/login)

Repeating the process from [Try to login again](https://id.fedoraproject.org/login) resulted in

> Nothing to Authenticate
> You will be redirected to this service when an application requires authentication against your Fedora Account
> Visit [Fedora Accounts](https://accounts.fedoraproject.org/) to create or modify your Fedora Account

On [Fedora Discussion](https://discussion.fedoraproject.org/) I arrived at a similar page on [Ipsilon](https://id.fedoraproject.org) but with error

> 400 - Bad Request
> Invalid transaction id
> If the error persists and if you think this is an error, contact Fedora Infra to resolve the problem.

Visiting [Fedora Discussion](https://discussion.fedoraproject.org/) I was not already identified, so clicked \[Log in\] and was non-interactively logged in. This non-interactive login is repeatable, i.e. if I log out, then click \[Log in\], I am logged in. The same is true of [Pagure](https://pagure.io/fedora-infrastructure/issues), where I get the "Authentication failure" or 400 error, but returning to the page and clicking \[Log in\]  logs me in non-interactively .

In summary: logging in to a Fedora service other than Accounts with a good password and OTP results in one or more errors from [Ipsilon](https://id.fedoraproject.org), but returning to the service and attempting to log in again results in a non-interactive login.

**Comment by ryanlerch** [comment]
Ok -- there seems to be some issue with longer passwords.

Will need to do some digging to see if this only happens when OTP is turned on.

**Comment by ryanlerch** [comment]
Duplicate of #1103
